api/src/migration/migrations/1735836154221-v-2-2-0.migration.ts

```diff
@@ -772,6 +772,42 @@ const migrateAndPopulateAttachmentMessages = async ({
   }
 };
 
+const addDefaultStorageHelper = async ({ logger }: MigrationServices) => {
+  const SettingModel = mongoose.model<Setting>(Setting.name, settingSchema);
+  try {
+    await SettingModel.create({
```

Comment:
I think that we can check if this setting exists or not before the creation otherwise we can get this setting duplicated.
![image](https://github.com/user-attachments/assets/62ae47b6-ecf0-46a4-8191-c2e6b642e0b8)

Reply:
Done